deploy/Jenkinsfile

```diff
@@ -282,5 +286,35 @@ pipeline {
                 }
             }
         }
+
+        stage('Post-Mortem') {
+            parallel {
+                stage('Documentation') {
+                    steps {
+                        sh "ls"
```

Comment:
if this you want to tackle later and if post-mortem stuff is ready, go ahead with merge